File: brewdesk-ui/src/types/beverage.ts
export interface Beverage {
  id: number
  name: string
  icon: string
  active: boolean
}

export interface BeverageSummary {
  beverageName: string
  icon: string
  count: number
}

export interface CreateBeverageRequest {
  name: string
  icon: string
}

export interface UpdateBeverageRequest {
  name: string
  icon: string
  active: boolean
}

File: brewdesk/src/main/java/com/office/brewdesk/entity/Beverage.java
package com.office.brewdesk.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "beverages")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Beverage {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false, unique = true)
    private String name;

    private String icon;

    @Builder.Default
    private boolean active = true;
}